src/test/java/com/wikia/webdriver/testcases/infoboxbuilder/InfoboxBuilderTests.java

```diff
@@ -12,6 +12,7 @@
 import com.wikia.webdriver.pageobjectsfactory.pageobject.PortableInfobox;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.special.themedesigner.SpecialThemeDesignerPageObject;
 
+import org.openqa.selenium.Dimension;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.openqa.selenium.Dimension'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)